spec/notifications/flag_notification_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe FlagNotification do
+  subject(:flag_notification) { described_class }
+  let(:flag) { create(:flag) }
+  let(:recipient) { create(:user) }
+
+  describe 'database delivery' do
+    it 'allows a valid FlagNotification to be saved' do
+      expect { flag_notification.with(flag: flag).deliver(recipient) }.to change { Notification.count }.by(1)
```

Comment:
Can change the `by(1)` at the end to `.from(0).to(1)` please? 

It's just a bit more declarative and will tell us straight away if notification objects are hanging around from other tests.